javascript/Spread_operator.md

```diff
@@ -0,0 +1,198 @@
+﻿# Spread Operator in javascript
+
+## Projected time
+
+About 30-40 minutes
+
+- 15 min for spread operator
+- 25 min for examples
+
+## Spread operator
+Spread operator allows an iterable such as strings, array and sets to spread or expand individually inside a receiver.
+
+The **syntax** of spread operator is `var variable  = [...value]`.
+
+## Why it is useful?
++ We can merge two objects together using spread operator.
+Example:
+```
+let obj1 = {a: 1, b: 2, c: 3}
+let obj2 = {d: 4, e: 5, f: 6}
+let merge = {...obj1, ...obj2} // merge is {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6}
+```
++ Spread operator with object is used to make copy of existing object or to make a new object with properties.
```

Comment:
I love these examples!  Can you: A. tell them to try it in the console or IDE or B. give the answer in a comment for all of them like line 21?

Reply:
Include the answer here too, please.